Makefile

```diff
@@ -98,8 +98,7 @@ bandit:
 .PHONY: pip-audit
 pip-audit:
 	pip-audit --version
-	# TODO: Fix the issue with the vulnerable ecdsa and jose libraries
-	pip-audit -r requirements.txt --ignore-vuln GHSA-wj6h-64fc-37mp --ignore-vuln GHSA-cjwg-qfpm-7377 --ignore-vuln GHSA-6c5p-j8vq-pqhj
+	pip-audit -r requirements.txt
```

Comment:
Let's use OSV as a vulnerability service as it collects vulnerabilities from multiple sources:
```suggestion
	pip-audit --vulnerability-service osv -r requirements.txt
```